### PR TITLE
fixes #2239 addes suricata eve.json as input

### LINKED ISSUE #2239: Add Suricata Eve logging to Telegraf
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md

[x] I have searched the existing issues and I'm convinced that mine is new.

[x] When the request is meant for an existing plugin, I've added its name to the title.

**Is your feature request related to a problem? Please describe.**
I am not able to send the contents of /var/log/suricata/eve.json to influxdb via telegraf
**Describe the solution you'd like**
I would like to be able to use the tail input plugin using the sample config below
```
[[inputs.tail]]
  data_format = "json"
  files = ["/var/log/suricata/eve.json"]
  name_override = "suricata"               
  tag_keys = ["event_type","src_ip","src_port","dest_ip","dest_port"]
  json_string_fields = ["*"]
```

**Describe alternatives you've considered**
I have considered use the syslog plugin but that is not exposed in the gui and would require a multiple step pipeline

**Additional context**
I am building an opensource RMM and would like to display suricata alerts in a monitoring dashboards

### PR DESCRIPTION
**Pull request by tiny6996**

### CODE REVIEW

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/input.xml [hunk 1]
```diff
@@ -137,4 +137,10 @@
         <type>checkbox</type>
         <help>Can increase metric gather times.</help>
     </field>
+    <field>
+        <id>input.intrusion_detection_alerts</id>
+        <label>Intrusion Dectection Alerts</label>
+        <type>checkbox</type>
+        <help>Requires Intrustion detection alerts are stored locally </help>
```
Comment (mimugmail) [inline net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/input.xml:144]: Please end the sentence with a dot, not a space :) Rest looks good

**Comment by tiny6996** [comment]
fixed the spelling issue thanks for the quick feedback

**Review by mimugmail — APPROVED** [review]
Great, thx

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**